kernel/kernel-api/src/main/java/io/delta/kernel/internal/snapshot/SnapshotManager.java

```diff
@@ -272,13 +272,15 @@ private Optional<LogSegment> getLogSegmentFrom(
      * @return Some LogSegment to build a Snapshot if files do exist after the given
      * startCheckpoint. None, if the directory was missing or empty.
      */
-    private Optional<LogSegment> getLogSegmentForVersion(
```

Comment:
nit: can we make this protected scope, once the suite is moved to the same package.